v1/api/swagger/components/schemas/Host.yaml

```diff
@@ -191,3 +192,5 @@ allOf:
         description: Host health summary status
       Labels:
         $ref: ./Labels.yaml
+      UserLabels:
+        $ref: ./Labels.yaml  
```

Comment:
Just want to bring up to see if feasible to follow the standards - https://developer.greenlake.hpe.com/docs/greenlake/standards/api/naming/

<img width="676" alt="image" src="https://github.com/HewlettPackard/hpegl-metal-client/assets/65302544/907fc821-5dcc-467b-9a5b-587bb7d324fe">

Reply:
the recommendation in api standards is to use a single field to hold both types of tags.  Some thoughts, but leaving it up to you & leads. and my apologies for not noticing this earlier.
1. 'Labels' is an old implementation, at least can a new field be named 'Tags' ? 
2. Deprecate 'Labels' and use new field 'tags' for both purposes ( internally handle migration / read the value from both/support both for sometime) ?

Reply:
An answer to the question I had raised during the design review -- :(. 

- A reason for separating out system and user tags was that it is being stored as separate fields in the DB. This was to avoid having to filter out the system tags in every response.
- Since the operator API uses the DB model it made sense to keep them separated in the project API as well. Would have the same structure with either API.

I do not think I would want to add 'Tags' only for a new field. 

Would like additional feedback since the way forward with this would potentially require me to redo backend logic.

@mchuang3 @dbozzato81 @AnuBose

Reply:
I don't think a specific field can be deprecated. @chaitra-mylarappachar ?

Reply:
Yet to try, but from spec, there is an attribute for this.
<img width="762" alt="image" src="https://github.com/HewlettPackard/hpegl-metal-client/assets/65302544/70ed6d75-fc52-41d1-bc76-35441c3f29a4">

Reply:
I'm not 100% following the discussion. I think these used to be called "Labels", but now they're called "Tags"? Is that just a GUI change or is it what other teams are calling them? Our backend Host object model still have them defined as "Labels". In any case, I would not put in the energy to do schema migration and make them Tags on the Metal DB side. Sort of same argument about changing Hosts to Compute Instances & Projects to Compute Groups. Just not worth the effort. It's easy enough to keep comments clear that Labels and Tags are the same thing.
As far as what Project API uses or calls them, it's perfectly fine to call them Tags in the Project API, which is translated to Labels in the back end business logic handling code. I'm ok if you want to deprecate original "Labels" and define "Tags" and "UserTags" instead. I'm fine w/ Labels/UserLabels as well.

Reply:
@mchuang3
- I'm not sure what the history is on Labels vs Tags. Maybe 'Labels' was chosen since we already have a resource called 'Tags' (used with machines)? It is being referred to as tags (Rally, other teams, UX, ...)
- Project API: Apart from the Labels vs Tags, I think Chaitra's ask from the API standards perspective is to not have a new field for 'user tags'. Both system and user tags are relayed via the same field -- very similar to the way AWS tagging works.

I was trying to see if I could keep the tagging fields for Operator API and Project API as close as possible. I probably should relay system and user tags via the same/single field.

Reply:
How would backend differentiate between system/service vs. user tags if they're passed in with single field? I see backend has some keys defined with "hpe:gl:...". I think those are the system tags that users cannot modify, but where is that documented in the client API? Anything stopping user from creating a label/tag with "hpe:gl..." prefix accidentally and unable to change it later?

Reply:
Updating with offline conversations:
- System tags have the prefix 'hpe:' (defined in the GL Tagging standards). These tags are restricted by role, so cannot be mistakenly added. Service teams will follow the same standards and are currently doing this.
- Chaitra provided a PCE API perspective as well. They will most like funnel system and user tags via a single field as well. Even though there is a transformation point (PCE to Metal API) it makes sense to maintain a similar approach for Metal/Project API.

So, will deprecate Labels and go with a single 'Tags' field for both system and user tags.

Reply:
Looking at the changes needed with deprecating Labels and using Tags. Given that we really should changing API version with such a change and we do not currently do that. Also, having to support both Labels and Tags (until service teams switch completely to Tags) would result in backend having to send tags in both Labels and Tags fields. 

Will leave field as Labels and use this for both system and user tags.